db/schema.rb

```diff
@@ -11,11 +11,21 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151101132623) do
+ActiveRecord::Schema.define(version: 20151105221354) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "book_lists", force: :cascade do |t|
+    t.integer  "book_id",    null: false
```

Comment:
screenhero notes:

could add `foreign_key` to avoid the dependent destroy in model

https://robots.thoughtbot.com/referential-integrity-with-foreign-keys
http://edgeguides.rubyonrails.org/active_record_migrations.html#foreign-keys